libpod/image/image.go

```diff
@@ -274,6 +266,16 @@ func (i *Image) getLocalImage() (*storage.Image, error) {
 	if err == nil {
 		return repoImage, nil
 	}
+
+	hasReg, err := i.hasRegistry()
+	if err != nil {
+		return nil, errors.Wrapf(err, imageError)
+	}
+	// if the input name has a registry in it, the image isnt here
+	if hasReg {
+		return nil, errors.Errorf("%s", imageError)
+	}
+
 	return nil, errors.Wrapf(err, imageError)
```

Comment:
This line is now completely broken, because `err == nil`, and `errors.Wrapf(nil, …) == nil`.

Reply:
fixed